src/relay_pic/main.c

```diff
@@ -0,0 +1,162 @@
+/* 
+ * File:   pinConfigMain.c
+ * Author: zacha
+ *
+ * Created on March 1, 2020, 3:21 PM
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <xc.h>
+#include <pic16f1826.h>
+#include "mcc_generated_files/adc.h"
+#include "mcc_generated_files/mcc.h"
+
+typedef unsigned char uint8_t;
+typedef unsigned short uint16_t;
+
+#define POWER (1 << 2)
+#define SELECT (1 << 3)
+#define LIM1 (1 << 7)
+#define LIM2 (1 << 6)
+#define DIP1 (1 << 4)
+#define DIP2 (1)
+#define DIP3 (1 << 2)
+#define DIP4 (1 << 3)
+#define LED3 (1 << 5)
+#define CURR_SENSE1 (1 << 1)
+#define CURR_SENSE2 (1)
+
+#pragma config WDTE = OFF
+#pragma FOSC = INTOSC

```

Comment:
I know I mentioned `config.h` in a previous comment - whoops. Looking at the PR now, your generated MCC configuration already has `device_config.h` which already contains all of these configuration pragmas. The ones in `main.c` are unnecessary.